metarecord/tests/conftest.py

```diff
@@ -186,3 +187,11 @@ def attribute_group(choice_attribute):
     choice_attribute.group = group
     choice_attribute.save(update_fields=('group',))
     return group
+
+
+@pytest.fixture
+def tos_importer_excel_file_path():
+    excel_filename = "00_00_01_02.xlsx"
+    current_file = os.path.realpath(__file__)
+    current_directory = os.path.dirname(current_file)
+    return os.path.join(current_directory, "test_data", excel_filename)
```

Comment:
I meant that this should be done with toplevel code in the test module in question (`test_importer.py`):
 ```python
from pathlib import Path

MYDIR = Path(__file__).parent.absolute()
EXCEL_FILE = MYDIR / "test_data" / "00_00_01_02.xlsx"
```

But this is fine too.